Pod/Classes/Acknowledgements/AcknowledgementViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  AcknowledgementViewController.swift
+//  Pods
+//
+//  Created by Michael Skiba on 11/21/16.
+//
+//
+
+import UIKit
+
+open class AcknowledgementViewController: UIViewController {
+
+    public static let defaultLicenseFormatter: (String) -> NSAttributedString
+        = { string in
+            let font = UIFont.preferredFont(forTextStyle: .body)
+            let paragraphStyle = NSMutableParagraphStyle()
+            paragraphStyle.hyphenationFactor = 1
+            paragraphStyle.paragraphSpacing = font.pointSize / 2
+            let attributes: [String:Any] = [NSFontAttributeName: font,
```

Comment:
#pp space after `:`, each dictionary key/value pair on its own line with a comma after the last one.